addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/handler/FreeboxHandler.java

```diff
@@ -175,15 +189,15 @@ private boolean authorize(boolean useHttps, String fqdn, String apiBaseUrl, Stri
             loginManager.setAppToken(configuration.appToken);
             loginManager.openSession();
             return true;
-        } catch (FreeboxException | InterruptedException e) {
-            logger.error("{}", e.getMessage());
+        } catch (Exception e) {
+            logger.warn("Thing {}: error while opening a session", getThing().getUID(), e);
```

Comment:
Is this situation a bug? Or can we get here in "normal" operation (e.g. on network issues)? If so, setting the Thing status would be the better thing to do.